CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased](https://github.com/ruflin/Elastica/compare/7.1.5...master)
 ### Backward Compatibility Breaks
+* Removed `string` parameter type-hint to `value` in `SetProcessor::setValue`.
```

Comment:
Can you expand this a bit more and explain in which scenarios it is breaking? Basically only if someone inherits from SetProcessor